netlify/next-runtime
# feat: use display names for edge functions

### Summary

Edge functions now support display names. This PR adds support for these. Aside from the ipx and dev edge funcitons, these just use the name from Next.js, which is the path.

### Test plan

1. Visit a deploy that uses edge functions (e.g. [this one with middleware](https://app.netlify.com/sites/next-plugin-edge-middleware/edge-functions?scope=deployid:634004bb7ee81700098bf533) or [this one with server components](https://app.netlify.com/sites/next-plugin-rsc-demo/edge-functions?scope=deployid:634004bbd73ce8000850da97)
2. View the edge function logs
3. Observe that the list of functions in the filter uses display names (e.g. middleware, next/image handler)

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal

### Standard checks:




---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.
